src/control/server/mgmt_system.go

```diff
@@ -348,6 +349,72 @@ func (svc *mgmtSvc) PingRanks(ctx context.Context, req *mgmtpb.RanksReq) (*mgmtp
 	return resp, nil
 }
 
+// ReformatRanks implements the method defined for the Management Service.
+//
+// Reformat storage of data-plane instances (DAOS system members) managed by harness.
+func (svc *mgmtSvc) ReformatRanks(parent context.Context, req *mgmtpb.RanksReq) (*mgmtpb.RanksResp, error) {
```

Comment:
This name is also somewhat misleading, as the actual format happens via a second step. This would be more accurately named if it indicated that it's performing a preparation step.

Reply:
again reset format sounds better